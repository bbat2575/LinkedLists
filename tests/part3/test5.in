NEW 1
one
NEW 3
{0}
1
two
VIEW-NESTED +1
VIEW-NESTED -1
VIEW-NESTED +0
VIEW-NESTED -0
VIEW-NESTED +
VIEW-NESTED -
VIEW-NESTED 2
VIEW-NESTED o
VIEW-NESTED ZERO
VIEW-NESTED .
VIEW-NESTED 0.
VIEW-NESTED .0
VIEW-NESTED 0.0
VIEW-NESTED 0e0
VIEW-NESTED 2e-6
VIEW-NESTED 1 1
VIEW-NESTED  1
VIEW-NESTED   1
VIEW-NESTED
VIEW-NESTED  
  VIEW-NESTED
1VIEW-NESTED
1 VIEW-NESTED
VIEW-NESTED1
 VIEW-NESTED 1
  VIEW-NESTED 1
VIEW NESTED 1
VIEW- NESTED 1
VIEW -NESTED 1
V IEW-NESTED 1
VIEW-NESTE D 1
view-nested 1
View-nested 1
View-Nested 1
VIEW-nESTED 1
VEIW-NESTED 1
