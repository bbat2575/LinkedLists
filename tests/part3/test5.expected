
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: one

> Nested 1: {List 0} -> 1 -> two

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> {one} -> 1 -> two

> {one} -> 1 -> two

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: VIEW-NESTED

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: VIEW

> INVALID COMMAND: INPUT

> INVALID COMMAND: VIEW

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> 
